go/vt/mysqlctl/mycnf.go

```diff
@@ -153,8 +158,21 @@ func normKey(bkey []byte) string {
 
 // ReadMycnf will read an existing my.cnf from disk, and update the passed in Mycnf object
 // with values from the my.cnf on disk.
-func ReadMycnf(mycnf *Mycnf) (*Mycnf, error) {
+func ReadMycnf(mycnf *Mycnf, waitTime time.Duration) (*Mycnf, error) {
 	f, err := os.Open(mycnf.Path)
+	if waitTime != 0 {
+		timer := time.NewTimer(waitTime)
+		ticker := time.NewTicker(myCnfWaitRetryTime)
```

Comment:
We can go back to using a sleep for this as well (similar to the other PR).

Reply:
Done